packages/cdktf-cli/templates/csharp/Program.cs

```diff
@@ -5,13 +5,8 @@
 
 namespace MyCompany.MyApp
 {
-    class MyApp : TerraformStack
+    class MyApp : Program
```

Comment:
The class name should be `Program` and it won't extend/implement anything.

Reply:
Also creating an instance of this below will fail since that constructor doesn't exist.